src/tpu.rs

```diff
@@ -225,6 +224,12 @@ impl Tpu {
         // TODO: Fix BankingStage/BroadcastService to operate on `slot` directly instead of
         // `max_tick_height`
         let max_tick_height = (slot + 1) * leader_scheduler.read().unwrap().ticks_per_slot - 1;
+        let blob_index = blocktree
+            .meta(slot)
+            .expect("Database error")
+            .map(|meta| meta.consumed)
+            .or(Some(0))
```

Comment:
oops, `unwrap_or(0)`

Reply:
Ok now I think you're just trying to get ahead of me in CI

Reply:
ha!